backend/apps/readings/analysis.py

```diff
@@ -32,3 +75,28 @@ def total_view_time(self):
             for view_time in response.get_parsed_views():
                 total_view_time += view_time
         return total_view_time
+
+    def all_responses(self):
```

Comment:
If you pull the latest version of master, you'll see that I've sketched in a test case for the webapp version of the analysis stuff. Please add a test for this analysis function.

Reply:
docstring plz!